WooCommerce/src/main/kotlin/com/woocommerce/android/ui/bookings/compose/BookingSectionHeader.kt

```diff
@@ -1,21 +1,36 @@
 package com.woocommerce.android.ui.bookings.compose
 
 import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 
 @Composable
-fun BookingSectionHeader(@StringRes header: Int) {
-    Text(
-        text = stringResource(header),
-        fontWeight = FontWeight.Medium,
-        style = MaterialTheme.typography.titleMedium,
-        modifier = Modifier.padding(start = 16.dp, top = 16.dp, end = 16.dp, bottom = 8.dp)
-    )
+fun BookingSectionHeader(
+    @StringRes header: Int,
+    action: @Composable () -> Unit = {},
+    modifier: Modifier = Modifier
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
```

Comment:
```suggestion
        verticalAlignment = Alignment.CenterVertically,
        horizontalArrangement = Arrangement.spacedBy(16.dp),
```

np: I’m not sure whether the FROM or TO text will be too long in any case, but just in case, let’s add 16dp spacing between them.

Reply:
Done